src/reducers/createAppReducer.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import { CREATE_APP } from '../constants';
+
+export default function userReducer(
```

Comment:
Please use reducer generator here - Refer `createRequestReducer` in batteries reducer utils.

Reply:
Also, on `success`, we should update the `apps` layer in the redux store with this `appName` and `appId`.